_posts/2019-10-28-MongoDB.md

```diff
@@ -38,17 +38,12 @@ Before doing so, you need to enable Access Control. By enabling access control,
 $ mongo
 ```
  
-*NB: You probably need to add a **<mark>port--</mark>** and **<mark>host--</mark>** after the mongo command depending on how you set up your deployment*.
+*NB: You probably need to add a **```port--```** and **```host--```** after the mongo command depending on how you set up your deployment*.
```

Comment:
So some pointers here:

* When you use code snippets in-line you use a single ` mark.
* Don't combine code snippet with bold